### ISSUE TITLE
Stats and difference are not aligned on iPad

### ISSUE THREAD

**Issue by mitogh** [google/site-kit-wp#4582]
## Bug Description

When visiting the main dashboard the numbers from the report and the difference are not aligned on the same page due some titles being wrapped in 2 lines.

Related from:

- #4454 

## Steps to reproduce

1. Go to the dashboard of the page.
2. Make sure the Google Analytics module is set up or set it up.
3. Scroll down to the metrics area and observe the result.
4. See numbers and diff values are not aligned properly.

## Screenshots

![2021-12-17_16-56](https://user-images.githubusercontent.com/3921289/146617358-b7f6a578-8376-43ef-8e55-fc24c2485527.png)

## Additional Context

 - PHP Version: 7.4
 - OS: iOS
 - Browser: Chrome
 - Plugin Version: 1.48.0
 - Device: iPad

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by aaemnnosttv** [comment]
Thoughts on this one @felixarntz ?

**Comment by felixarntz** [comment]
Hmm, this would be nice to fix for sure - not highest priority but worth tackling. I'm not sure on the best way to address this via CSS (maybe some sort of table-like layout?), but I think this should behave in a way that all three parts of each `DataBlock` align vertically with the same part in the other `DataBlock`s around it. Potentially this will justify introducing a `DataBlocks` wrapper component to handle the styling. I'll add ACs.

**Comment by asvinb** [comment]
@aaemnnosttv @felixarntz This regression was introduced while fixing https://github.com/google/site-kit-wp/issues/4492 (check the screenshot in the QA brief), from this PR: https://github.com/google/site-kit-wp/commit/82dae44ebdbeeddeb7c1565c22f146ebd8ebf282#diff-2b68f8b75627d76765896f560d8ac21475b2f337343bb3a29289a9aa5bda4d7d. If we revert that change, the widget will look like :
![image](https://user-images.githubusercontent.com/31818/149532706-64243a66-82e0-4005-b9e3-6946bb82261d.png)
with the numbers and text aligned. We can revert that change and apply some specific styles to the `SearchFunnelWidget` widget to cater for the alignment when `analytics` is not connected.

Let me know what you think and i'll write the IB for it.

**Comment by felixarntz** [comment]
@asvinb Good catch! Yeah, I think just reverting wouldn't be right since then we would bring back the one "bug" outlined in #4492. Your idea sounds solid, let's fix this in a way that also keeps the visual fix from #4492 in place, for when Analytics is not active.

**Comment by tofumatt** [comment]
Nice one @asvinb, that looks good. 👍🏻 

IB ✅

**Comment by wpdarren** [comment]
### QA Update: ❌ 

@nfmohit Thanks for the detailed QAB!  I have two observations, but they might be out of scope. Still, I thought the first one might be linked to the fix here. 

1. On all iPads (seems fine on mobile phones) there isn't any padding around the `Unique visitors from search` title when you click on it to get the graph data, see screenshot below.  The title is tight up to the border. 

![image](https://user-images.githubusercontent.com/73545194/170165731-ffcc1817-6620-41ff-9c3c-59db550b49ca.png)

2. When Analytics is disconnected, and the screen size is around 601 to 684 pixels wide, the `total impressions ` title and data is misaligned.  Again, not 100% sure if this is an impact of the change. 

Interested in your thoughts.

**Comment by wpdarren** [comment]
### QA Update: ✅ 

**Verified:**

- The initial styling issue on all iPad sizes for `Unique visitors from search ` 
- When disconnecting the Analytics module the DataBlock content in the Search Funnel widget is aligned to the top condition.
- When the Goals CTA is shown, and the data blocks aren't top-aligned, the layout does not break. 
- The issue reported earlier has been fixed when Analytics is disconnected. The `total impressions` data block does not break and looks good on all iPad sizes. 

<details>
 <summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/73545194/173974652-aef95ef2-71f8-45ec-97d4-d7249282cb0a.png)
![image](https://user-images.githubusercontent.com/73545194/173974669-58a07174-153d-4105-9686-bba3a6fc6f1b.png)
![image](https://user-images.githubusercontent.com/73545194/173974682-d77e3fd6-a2c2-40b1-b43c-6de3c8317114.png)

</details>
